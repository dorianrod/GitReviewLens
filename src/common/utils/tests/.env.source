GIT_BRANCHES=[{"name":"master","repository":{"organisation":"","project":"facebook", "name": "lexical", "url": "git@github.com:facebook/lexical.git", "token": "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"}}]

DEBUG=0
VERBOSE=0

WHATEVER="toto"
