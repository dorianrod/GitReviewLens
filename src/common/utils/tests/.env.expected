GIT_BRANCHES=[{'test':'toto'}]

DEBUG=0
VERBOSE=0

WHATEVER="toto"
